### PR TITLE
content: ockam c coding standard md file

### PR DESCRIPTION
**Pull request by mwiszniewski**
version 1.0

### CODE REVIEW

**Comment by miedziak** [comment]
Maybe we could split this documentation into multiple files, so it would show in left sidebar as multiple smaller chapters

**Comment by mattgreg** [comment]
I think that the 'smaller chapters' is already taken care of by how we surface H2s (## in markdown) on the right menu.

File: src/content/learn/guides/Contributing/Ockam_C_Coding_Standard.md [hunk 1]
```diff
@@ -0,0 +1,879 @@
...
+A name is the result of a long deep thought process about the ecology it lives in. Only a
+programmer who understands the system as a whole can create a name that "fits" with the
+system. If the name is appropriate everything fits together naturally, relationships are clear,
+meaning is derivable, and reasoning from common human expectations works as expected.
+If you find all your names could be Thing and DoIt then you should probably revisit your design.
+
+### Function Names
+
+
+● Usually every function performs an action, so the name should make clear what it does:
```
Comment (mattgreg) [inline src/content/learn/guides/Contributing/Ockam_C_Coding_Standard.md:19]: Please remove the 'dot' icon and use the bulleted list markdown style here.

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists

Comment (mwiszniewski) [reply src/content/learn/guides/Contributing/Ockam_C_Coding_Standard.md:19]: > Formatting is incorrect in many places. Bullets aren't put on separate lines, and indentations are incorrect. Please spend a bit more time and make it look as much like the original as possible.

There was some encoding issue. Now it should look fine.

**Comment by mwiszniewski** [comment]
> The path on this file has changed.
> 
> Originally:
> /_C_ontributing/Ockam_C_Coding_Standard.md
> 
> Changed to:
> 
> /_c_ontributing/Ockam_C_Coding_Standard.md
> 
> the 'C' was changed to a lower case 'c'

I changed directory name.

**Comment by mattgreg** [comment]
@mwiszniewski 
This PR should only have the single file with the coding standard. Can you remove all of the other file changes? 

Then can you squash the commit history?

**Pull request closed without merging**